appinventor/common/src/com/google/appinventor/common/version/AppInventorFeatures.java

```diff
@@ -148,4 +148,8 @@ public static String chatBotHost() {
     return "https://chatbot.appinventor.mit.edu/";
   }
 
+  public static boolean doingSurvey() {
+    return true;
```

Comment:
Don't we want to make this false by default and then turn it on in the branding branch?